modules/nw-route-set-or-delete-http-headers.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * networking/route-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-route-set-or-delete-http-headers_{context}"]
+= Setting or deleting HTTP request and response headers in a route
+
+You can set or delete certain HTTP request and response headers for compliance purposes or other reasons. You can set or delete these headers either for all routes served by an Ingress Controller or for specific routes.
+
+For example, you might want to enable a web application to serve content in alternate locations for specific routes if that content is written in multiple languages, even if there is a default global location specified by the Ingress Controller serving the routes.
+
+The following procedure creates a route that sets the Content-Location HTTP request header so that the URL associated with the application, `\https://app.example.com`, directs to the location `\https://app.example.com/lang/en-us`. Directing application traffic to this location means that anyone using that specific route is accessing web content written in American English.
+
+.Prerequisites
+* You have installed the OpenShift CLI (`oc`).
+* You are logged into an {product-title} cluster as a project administrator.
+* You have a web application that exposes a port and an HTTP or TLS endpoint listening for traffic on the port.
+
+.Procedure
+. Create a route definition and save it in a file called `app-example-route.yaml`:
++
+.YAML definition of the created route with HTTP header directives
+[source,yaml]
+----
+apiVersion: route.openshift.io/v1
+kind: Route
+# ...
+spec:
+  host: app.example.com
+  tls:
+    termination: edge
+  to:
+    kind: Service
+    name: app-example
+  httpHeaders:
+    actions: <1>
+      response: <2>
+      - name: Content-Location <3>
+      action:
+        type: Set <4>
+        set:
+          value: /lang/en-us <5>
```

Comment:
All the "action:" should move back two spaces, so a of the "action:" is aligned with n of the above "- name:"

Reply:
"type:", "set:" and "value:" under "action:" should also be moved back two spaces.

Reply:
fixed